### PR TITLE
(IMAGES-831) June 2018 Patch Tuesday Windows Refresh

### PR DESCRIPTION
**Pull request by jcoconnor**
June 2018 Patch Tuesday refresh with a number of the May 2018 changes reverted.

1. Revert to Cygwin 2.6.0 because of build problems.
2. Remove IPV6 Disable as it appears to break some facter tests (IMAGES-861)
3. Present Windows-10 Build 1803 as a new build to allow IMAGES-859 to be investigated.

### CODE REVIEW

File: templates/win/10-1803/x86_64/files/slipstream-filter [hunk 1]
```diff
@@ -0,0 +1 @@
+#
```
Comment (RandomNoun7) [inline templates/win/10-1803/x86_64/files/slipstream-filter]: ?

Comment (michaeltlombardi) [inline templates/win/10-1803/x86_64/files/slipstream-filter]: wat

Comment (jcoconnor) [reply templates/win/10-1803/x86_64/files/slipstream-filter]: Just an empty file that needs to be there but is mostly unused.

File: templates/win/10-ent/i386/vars.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
     "windows_version"   : "Windows-10",
     "image_name"        : "Windows 10 Enterprise",
     "product_key"       : "NPPR9-FWDCX-D2C8J-H872K-2YT43",
-    "version"           : "20180608_DEV",
-    "iso_url"           : "https://artifactory.delivery.puppetlabs.net/artifactory/generic/iso/windows/en_windows_10_business_editions_version_1803_updated_march_2018_x86_dvd_12063341.iso",
+    "version"           : "20180615_PROD",
+    "iso_url"           : "https://artifactory.delivery.puppetlabs.net/artifactory/generic/iso/windows/en_windows_10_multi-edition_vl_version_1709_updated_dec_2017_x86_dvd_100406182.iso",
```
Comment (RandomNoun7) [inline templates/win/10-ent/i386/vars.json]: Is this supposed to be 1709?

Comment (RandomNoun7) [reply templates/win/10-ent/i386/vars.json]: nm

Comment (jcoconnor) [reply templates/win/10-ent/i386/vars.json]: Yes - thats what the last windows 10 rollout was.

**Pull request merged**